driver/registry_default_registration.go

```diff
@@ -28,6 +28,11 @@ func (m *RegistryDefault) PostRegistrationPostPersistHooks(ctx context.Context,
 		initialHookCount = 1
 	}
 
+	if credentialsType == identity.CredentialsTypeCodeAuth && m.Config().SelfServiceCodeStrategy(ctx).RegistrationEnabled {
+		b = append(b, m.HookCodeAddressVerifier())
```

Comment:
yes, since you might have multiple identifiers with the code extension. Only one of those emails can be used to register with, and the other will be marked as unverified. Once logging in with the other, this will mark the address as verified.